src/components/Searcher.js

```diff
@@ -994,8 +1031,9 @@ class Searcher extends React.Component<SearcherDefaultProps, SearcherProps, Sear
     if (!facet) {
       return;
     }
+    const docIDSuffix = (weight === 1) ? '-add' : '-remove';
```

Comment:
I would change the delimiter from "-" to something that's unlikely to appear in a facet filter like a pipe "|" character... This will make it easier to split later to get the filter string back...